core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1719,6 +1709,12 @@ public String toString() {
       .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
       .setScope(Scope.WORKER)
       .build();
+  public static final PropertyKey WORKER_MASTER_CONNECT_RETRY_TIMEOUT =
+      new Builder(Name.WORKER_MASTER_CONNECT_RETRY_TIMEOUT)
+          .setDescription("Retry period before workers give up on connecting to master")
```

Comment:
After this period, what happens?

Reply:
I think the worker will stop after the timeout. This is an existing property I just reordered.